assets/css/wpr-admin-common.css

```diff
@@ -1,25 +0,0 @@
-.rocket-plugins-error{overflow:hidden; padding-left:20px; list-style-type:disc}
-.rocket-plugins-error li{line-height:25px}
-#wp-admin-bar-purge-all-all .dashicons{font:normal 1.5em/1 'dashicons'; -webkit-font-smoothing:antialiased; vertical-align:middle; padding-right:3px; display:inline-block}
-.rocket-purchase, .dashicons.dashicons-update.rocket-dashicons:before{color:#D54E21}
-.rocket-purchase{font-weight:900}
-.rocket-renew{font-weight:400}
-.rkt-disabled{color:#CCC}
-.rkt-note{font-style:italic; font-weight:bold; font-size:smaller}
-#wp-admin-bar-wp-rocket #wp-admin-bar-purge-all .ab-item>img, #wp-admin-bar-wp-rocket #wp-admin-bar-preload-cache .ab-item>img{max-width: 16px; max-height: 11px}
-.rocket-promo-bubble {
-	display: inline-block;
-	vertical-align: top;
-	box-sizing: border-box;
-	margin: 1px 0 -1px 2px;
-	padding: 0 5px;
-	min-width: 18px;
-	height: 18px;
-	border-radius: 9px;
-	background-color: #00A66B;
-	color: #fff;
-	font-size: 11px;
-	line-height: 1.6;
-	text-align: center;
-	z-index: 26;
-}
```

Comment:
why is this deleted?

Reply:
@Tabrisrp I guess this is deleted as we weren't compiling non minified assets before they were a bit outdated.

Reply:
it's still css that we need it should not be removed

Reply:
locally it shows the file with content, not sure what GH is on, but we can ignore that for now